text/1017-ember-qunit-themes.md

```diff
@@ -0,0 +1,66 @@
+---
+stage: accepted
+start-date: 2024-03-28T00:00:00.000Z
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/1017/
+project-link:
+suite: 
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+# Update QUnit's Theme 
+
+## Summary
+
+Add [qunit-theme-ember](https://github.com/IgnaceMaes/qunit-theme-ember) to `ember-qunit`, so that the CSS is included by default.
+
+## Motivation
+
+The default qunit theme is very outdated.
+Adding a new theme will make testing feel fresh, and more relaxed.
+
+## Detailed design
+
+Finish this PR: https://github.com/emberjs/ember-qunit/pull/1166
+- This design will also allow the inclusion of multiple themes, for varying experiences, and preferences -- in general more experimentation so that we can later choose a _best_ theme.
+- The old (current before this PR) theme will still be chooseable, but won't be default
+
+Release as a major to be safe, in case folks are already customizing the existing qunit styles.
+
+
+## How we teach this
+
+Changelog would be fine for ember-qunit.
+
+## Drawbacks
+
```

Comment:
Having _one more_ dependency to manage could be seen as a drawback. If QUnit would do a new major changing the HTML structure of the test runner, the package would have to get updated to be compatible.

However, considering there basically hasn't been any change in 10 years, I don't expect this to be a real issue. And in case it would happen, updating should be a big effort as the package is just a ~100 lines of CSS. Alternatively, when theming becomes configurable, a switch to another package that is already compatible would be very easy.